public/app/plugins/datasource/loki/modifyQuery.ts

```diff
@@ -90,6 +90,33 @@ export function addLabelFormatToQuery(query: string, labelFormat: { originalLabe
   return addLabelFormat(query, logQueryPositions, labelFormat);
 }
 
+/**
+ * Removes all comments from query.
+ * It uses  LogQL parser to find all LineComments and removes them.
+ *
+ * @param query
```

Comment:
Is it required to specify \@params in dockblocks when using TypeScript? Asking because it can be considered redundant.

Reply:
Yeah, you are right that in this case it is absolutely redundant. I'll remove it. However in some cases it can be quite useful and in that case you are encouraged to add it there. The documentation on code comments can be find here: https://github.com/grafana/grafana/blob/main/contribute/style-guides/code-comments.md

Updated in: https://github.com/grafana/grafana/pull/53254/commits/cdf0c839e3e48fa18afb1b0cd7a04a852bfeaab4